plugins/parsers/value/README.md

```diff
@@ -25,14 +25,35 @@ as the parsed metric.
   data_type = "integer" # required
 ```
 
+### Metric name
+
+It is recommended to set `name_override` to a measurement name that makes sense
+for your metric, otherwise it will just be set to the name of the plugin.
+
+### Datatype
+
 You **must** tell Telegraf what type of metric to collect by using the
 `data_type` configuration option. Available options are:
 
-1. integer
-2. float or long
-3. string
-4. boolean
-
-**Note:** It is also recommended that you set `name_override` to a measurement
-name that makes sense for your metric, otherwise it will just be set to the
-name of the plugin.
+- `integer`: converts the received data to an integer value. This setting will
+             produce an error on non-integer data.
+- `float`:   converts the received data to a floating-point value. This setting
+             will treat integers as floating-point values and produces an error
+             on data that cannot be converted (e.g. strings).
+- `string`:  outputs the data as a string.
+- `boolean`: converts the received data to a floating-point value. This setting
+             will produce an error on any data except for `true` and `false`
+             strings.
```

Comment:
hmmm this uses parsebool, which should return a bool again not a float right?

Reply:
Float, bool what's the difference? ;-P Waiting for the copy-and-paste AI to become more powerful. :-)